roles/idrac_gather_facts/molecule/secureboot/converge.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Converge idrac_gather_facts for SecureBoot
+  hosts: all
+  gather_facts: false
+  vars:
+    hostname: "{{ lookup('env', 'hostname') }}"
+    username: "{{ lookup('env', 'username') }}"
+    password: "{{ lookup('env', 'password') }}"
+    validate_certs: false
+    target:
+      - SecureBoot
+    idrac_gather_facts_uri_method: "GET"
+    idrac_gather_facts_uri_headers:
+      Accept: "application/json"
+      Content-Type: "application/json"
+      OData-Version: "4.0"
+    idrac_gather_facts_uri_body_format: "json"
+    idrac_gather_facts_uri_status_code:
+      - 200
+      - 400
+      - 401
+      - 404
+      - -1
```

Comment:
just checking if -1 is also a valid status code for the idrac api?

Reply:
Yes, it might occur if connection was aborted before a complete request.